### PR TITLE
quiz page added

### PR DESCRIPTION
**Pull request by palash23jain**
Screenshot 2022-02-21 at 9 28 49 AM

### CODE REVIEW

File: quiz-page/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<body>
+    <div class="navbar-container">
+        <div class="navbar-title">
+            <a href="/index.html">Cric-Quiz</a>
+        </div>
+        <input type="text" placeholder="search" class="navbar-search-input">
+        <div class="navbar-items">
+            <a href="/login/index.html" class="btn btn-warning">Login</a>
+        </div>
+    </div>
```
Comment (pritam-kr) [inline quiz-page/index.html:23]: use semantic HTML

**Review by pritam-kr — APPROVED** [review]
Good to go. You can merge it.

**Review by mimansha-swarup — APPROVED** [review]
good code structure
using var for most of the stuff 
great man 🔥

**Review by nshanbhag20001412 — APPROVED** [review]
Looks good! Good to go!

**Pull request merged**